micro-s3/src/main/java/com/aol/micro/server/s3/manifest/comparator/S3ManifestComparator.java

```diff
@@ -215,7 +215,7 @@ public synchronized boolean load() {
             data = oldData;
             versionedKey = oldKey;
             modified = oldModified;
-            logger.debug(e.getMessage(), e);
+            logger.info(e.getMessage(), e);
```

Comment:
newKey should be in the error message already, it just at too low a level to be logged.